app/src/pages/inside/settingsPage/demoDataTab/demoDataTab.scss

```diff
@@ -0,0 +1,127 @@
+// MOBILE styling:
+.demoDataTabContainer {
+  padding: 20px 10px;
+  display: flex;
+  flex-direction: column;
+
+  .youCanGenerateDat {
+    font-size: 14px;
+    line-height: 1.57;
+    text-align: left;
+    color: $COLOR--charcoal-grey;
+  }
+
+  .theSystemWillGene {
+    margin-top: 15px;
+    font-size: 12px;
+    font-weight: 600;
```

Comment:
Do not use  font-weight rule ever. If you need bold font, define it in font-family rule ( $FONT-SEMIBOLD )